io_scene_niftools/modules/nif_export/collision/havok.py

```diff
@@ -173,7 +173,8 @@ def export_bhk_rigid_body(self, b_obj, n_col_obj):
         n_r_body.restitution = b_r_body.restitution
         n_r_body.max_linear_velocity = b_obj.nifcollision.max_linear_velocity
         n_r_body.max_angular_velocity = b_obj.nifcollision.max_angular_velocity
-        n_r_body.penetration_depth = b_obj.collision.permeability
+        # no longer available in blender 3.1
```

Comment:
Probably want to stick a TODO on this if we need to map it to something else.

Reply:
> Probably want to stick a TODO on this if we need to map it to something else.

Does this break on export?

Reply:
Added. Rigid body support is minimal right now anyway.

Reply:
Marker is good enough for now 👍